omegaconf/basecontainer.py

```diff
@@ -769,3 +778,39 @@ def _update_types(node: Node, ref_type: type, object_type: Optional[type]) -> No
         if new_ref_type is not Any:
             node._metadata.ref_type = new_ref_type
             node._metadata.optional = new_is_optional
+
+
+def _instantiate_structured_config_impl(
+    conf: "DictConfig", instance_data: Dict[str, Any]
+) -> Any:
```

Comment:
should this be a method on DictConfig?

Reply:
I don't see why not.

Reply:
seems to make more sense there. move it?

Reply:
https://github.com/omry/omegaconf/pull/502/commits/2cf460f9b2187369bce8576ea30d67ff7920ceca